src/components/molecules/card/card.html

```diff
@@ -0,0 +1,13 @@
+<div class="card">
+  <div class="card-img">
+    <p class="title">class-title</p>
+    <div class="triangle"></div>
+    <div class="img">

```

Comment:
```suggestion
    <div class="img-wrapper">
```
クラス名がimgだと、コードを読む人に、 `img`要素 かな　という誤解を抱かせる可能性があるので、何かを囲む要素のクラス名には `wrapper` `wrap` `container` 等の単語を含めるといいと思います。